sql/spell_list.sql

```diff
@@ -860,7 +860,7 @@ INSERT INTO `spell_list` VALUES (880,'drain_iii',0x00000000000000630000000000000
 -- INSERT INTO `spell_list` VALUES (892,'luminohelix_ii',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1000,0,0,1.00,0,0,0,0,NULL);
 -- INSERT INTO `spell_list` VALUES (893,'full_cure',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1000,0,0,1.00,0,0,0,0,NULL);
 -- INSERT INTO `spell_list` VALUES (894,'refresh_iii',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1012,0,0,1.00,0,0,0,0,NULL);
--- INSERT INTO `spell_list` VALUES (895,'temper_ii',0x00000000000000000000000000000000000000000000,8,7,0,1,0,0,2000,48000,0,0,288,1000,0,0,1.00,0,0,0,0,NULL);
+INSERT INTO `spell_list` VALUES (895,'temper_ii',0x00000000630000000000000000000000000000000000,6,142,7,0,1,34,72,7000,36000,0,0,120,2000,0,0,1.00,1,300,0,0,'SOA');
```

Comment:
Where did you get this data from, if I may ask?

Concretelly, the spell animation.

Reply:
Oh I just copied the one from temper and changed the values. Never considered that the animation could be different

Reply:
![image](https://github.com/LandSandBoat/server/assets/60417494/c671f550-25c2-43ad-884c-b512a10f1e55)
it's 43

Reply:
Whoops, wrong packet. 1011.
![image](https://github.com/LandSandBoat/server/assets/60417494/cf166c77-ad16-40b3-82ef-6605ef532bbd)

Reply:
okay fixed